docs/hfvqe/odd_qubit.ipynb

```diff
@@ -1,948 +0,0 @@
-{
- "cells": [
```

Comment:
Did you mean to delete this file? If so, also remove entry from _toc.yaml

Reply:
yes. deleted from toc; added molecular data entry, which I had missed